fe/fe-core/src/main/java/com/starrocks/system/HeartbeatResponse.java

```diff
@@ -98,8 +102,7 @@ public static HeartbeatResponse read(DataInput in) throws IOException {
 
     @Override
     public void write(DataOutput out) throws IOException {
-        Text.writeString(out, type.name());
-        Text.writeString(out, status.name());
+        Text.writeString(out, GsonUtils.GSON.toJson(this));
```

Comment:
You changed the format. The new version process won't be compatible with old format. Is this OK?